src/add_forward_poly_association_plugin.ts

```diff
@@ -37,18 +39,21 @@ export const addForwardPolyAssociation = (builder: SchemaBuilder, option: Option
     // Find  all the forward relations with polymorphic
     const forwardPolyRelationSpec = (
       <PgPolymorphicConstraints>pgPolymorphicClassAndTargetModels)
-      .filter(con => con.from === table.id)
+      .filter(con => canonical(con.from) === canonical(table.id))
```

Comment:
why do you need to apply the canonical to the table.id ? I thought the function just converts `XXX:Model` to `Model`. So it should only apply to the model name ?

Reply:
You probably know more than I do about how this works, I just identified the parts I needed to change in order to "make it work" on my side through trial, error, and copious logging & debuggering 😬.